### PR TITLE
Get User Devices by Username - Generic - fixed an issue where Core didn't run in XSIAM

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
Related to: https://jira-dc.paloaltonetworks.com/browse/CIAC-8220

## Description
Fixed an issue where Cortex - Core - IR did not run in XSIAM, because of the different name of the integration and the different outputs.

### CODE REVIEW

**Comment by idovandijk** [comment]
@tomer-pan I added you as a reviewer since you are familiar with the playbooks :)

File: Packs/CommonPlaybooks/Playbooks/playbook-Get_User_Devices_by_Username_-_Generic.yml [hunk 1]
```diff
@@ -2128,6 +2125,194 @@ tasks:
...
+    task:
+      id: 1497e644-070b-440d-8a6a-5d238a793c8c
+      version: -1
+      name: Set XDREndpoints
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.11/Cortex-XSOAR-Administrator-Guide/Automations
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Get_User_Devices_by_Username_-_Generic.yml:2287]:
```suggestion
        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.12/Cortex-XSOAR-Administrator-Guide/Automations
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Get_User_Devices_by_Username_-_Generic_README.md [hunk 2]
```diff
@@ -67,158 +67,158 @@ This playbook does not use any sub-playbooks.
...
+| ServiceNow.Computer.Cost | Computer cost. | unknown |
+| ServiceNow.Computer.Comments | Computer comments. | unknown |
+| UserDevices | Devices retrieved by this playbook. | unknown |
+| UserDevices.Name | Devices names retrieved by this playbook. | unknown |
+| UserDevices.SerialNumber | Devices serial numbers retrieved by this playbook. | unknown |
+| UserDevices.ID | Devices IDs retrieved by this playbook. | unknown |
+| UserDevices.Model | Devices models retrieved by this playbook. | unknown |
+| UserDevices.OS | Devices operating systems retrieved by this playbook. | unknown |
+| UserDevices.Integration | The integration which retrieved the devices. | unknown |
+| UserDevices.MACAddress | Devices MAC Address retrieved by this playbook. | unknown |
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Get_User_Devices_by_Username_-_Generic_README.md:221]:
```suggestion
| UserDevices.MACAddress | Devices MAC address retrieved by this playbook. | unknown |
```

File: Packs/CommonPlaybooks/ReleaseNotes/2_4_15.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Get User Devices by Username - Generic
+
+- Fixed an issue where Cortex Core - Investigation & Response did not run and retrieve the devices in XSIAM.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_4_15.md:6]:
```suggestion
Fixed an issue where Cortex Core - Investigation & Response did not run and retrieve the devices in Cortex XSIAM.
```

**Comment by ShirleyDenkberg** [comment]
@michalgold @tomer-pan @AdiPeret Doc review completed.

**Review by tomer-pan — APPROVED** [review]

**Pull request merged**